test/LootBoxController.test.js

```diff
@@ -56,6 +56,12 @@ describe('LootBoxController', () => {
     let lootBoxControllerResult = await deployments.get('LootBoxController')
```

Comment:
Above this line- let's replace all of the `deployContract` calls with code that pulls in the already deployed ERC721, ERC20 and ERC1155